### DISCUSSION TITLE
WordPress logout without SSO Logout

### DISCUSSION THREAD

**Discussion by Stonebkfly** [oidc-wp/openid-connect-generic#483] [Q&A]
Out admins can impersonate a SSO user and then go to the WordPress site. In this case we would like to give them the option to just logout of the WordPress site and not the SSO. Is there a why we can do a local logout of WordPress without also logging out of the SSO?

**Comment by timnolte** [comment]
@Stonebkfly do you have the IDP logout URL populated in the plugin settings? I'll have to double check this myself that not having that filled out performs a standard WordPress logout. However, are you looking for the ability to keep regular users set to logout from the IDP but admins can logout only from WordPress?

**Reply by Stonebkfly** [reply]
We are adding a "Stop Impersonation" button to the menu when WordPress sees you are impersonating a user (via the AMR claim). Currently this button just log you out of WordPress and the IDP (default behavior). We already have a StopImpersonating endpoint at the IdP. So, I would like to log the user out of WordPress and then send them to this endpoint. Everything else should remain the same. Ie. Logouts should log the user out of WordPress and IdP as it currently does. Thank you Tim!

**Reply by Stonebkfly** [reply]
I have the "End Session Endpoint URL" populated
